src/services/upload/queueProcessor.ts

```diff
@@ -50,15 +54,14 @@ export default class QueueProcessor<T> {
     public async processQueue() {
         while (this.requestQueue.length > 0) {
             const queueItem = this.requestQueue.pop();
-            let response = null;
-
+            let response: any;
             if (queueItem.isCanceled.status) {
                 response = null;
             } else {
                 try {
                     response = await queueItem.request(queueItem.canceller);
                 } catch (e) {
-                    response = null;
+                    // ignore
```

Comment:
What are the cases when we will get exception and why are we ignore it instead of handling/logging it?

Reply:
all the exception are handled at the request level and logged there ( as this function can't bubble up the error to caller) so no error should come here because...but ya i should log the error and then ignore it